tests/loggers/test_csv.py

```diff
@@ -95,7 +94,6 @@ def test_file_logger_log_hyperparams(tmpdir):
         "bool": True,
         "dict": {"a": {"b": "c"}},
         "list": [1, 2, 3],
-        "namespace": Namespace(foo=Namespace(bar="buzz")),
         "layer": torch.nn.BatchNorm1d,
```

Comment:
This is allowed with `full_load` but not `safe_load`.

Security is at odds with functionality here.